project-addons/flask_middleware_connector/data/parameters.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo noupdate="1">
-    <record id="translations_to_export" model="ir.config_parameter">
-        <field name="key">translations.to.export</field>
-        <field name="value"></field>
-    </record>
+<odoo>
+    <data noupdate="1">
```

Comment:
Este cambio, aunque no cause problemas, es innecesario a no ser que vayáis a meter también datos con noupdate="0"
https://www.odoo.com/documentation/11.0/reference/guidelines.html

> The tag ```<data>``` is only used to set not-updatable data with noupdate=1. If there is only not-updatable data in the file, the noupdate=1 can be set on the ```<odoo>``` tag and do not set a ```<data>``` tag.

Reply:
Lo puse por que ese parámetro nos daba problemas al hacer un update all en nuestro local. Nos decía que ya existía, obviamente, por que está creado de la primera vez. Le puse el data por probar y así ya no daba el error. Pero bueno, de todas formas lo quito, la documentación dice claramente que se puede poner en el odoo